tests/integration/test_storage_iceberg/test.py

```diff
@@ -2717,7 +2717,45 @@ def test_writes_drop_table(started_cluster, format_version, storage_type):
     )
     # drop should not delete user data
     assert len(files) > 0
-    
+
+
+@pytest.mark.parametrize("format_version", [1, 2])
+@pytest.mark.parametrize("storage_type", ["s3", "local", "azure"])
+def test_writes_schema_evolution(started_cluster, format_version, storage_type):
+    instance = started_cluster.instances["node1"]
+    spark = started_cluster.spark_session
+    TABLE_NAME = "test_bucket_partition_pruning_" + storage_type + "_" + get_uuid_str()
+
+    create_iceberg_table(storage_type, instance, TABLE_NAME, started_cluster, "(x Int32)", format_version)
+    assert instance.query(f"SELECT * FROM {TABLE_NAME} ORDER BY ALL") == ''
+
+    instance.query(f"ALTER TABLE {TABLE_NAME} MODIFY COLUMN x Int64;", settings={"allow_experimental_insert_into_iceberg": 1})
```

Comment:
Could be easier to just set `allow_experimental_insert_into_iceberg = 1` in users profile in config for all tests here, as I think there is no need to disable it for tests, while many tests need it enabled.